projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -171,6 +155,46 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
     @ViewChild(IgxForOfDirective, { read: IgxForOfDirective, static: true })
     protected virtDir: IgxForOfDirective<any>;
 
+    /**
+     * Set the combo's overlaySettings that control how the list of items is displayed.
+     *
+     * ```html
+     * <igx-combo [overlaySettings] = "customOverlaySettings"></igx-combo>
+     * ```
+     *
+     * ```typescript
+     *  const customSettings = { positionStrategy: { settings: { target: myTarget } } };
+     *  combo.overlaySettings = customSettings;
+     * ```
+     */
+    @Input()
+    public set overlaySettings(val: OverlaySettings) {
+        if (val) {
+            const newSettings = val;
+            if (val.positionStrategy) {
+                newSettings.positionStrategy = val.positionStrategy.clone();
+            }
+            Object.assign(this._overlaySettings, newSettings);
```

Comment:
Probably safer if you keep your defaults separate, rather than overriding them.
Ideally you won't need a setter.

Reply:
Should likely behave similar to the igxToggleAction (defaults, extend and target provision)